client/src/containers/Overview/index.jsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Calendar from 'react-github-contribution-calendar';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Container, Grid, Dropdown, Accordion } from 'semantic-ui-react';
+import Octicon, { Repo, Grabber, Fold, Unfold, RepoPush, GitPullRequest } from '@primer/octicons-react';
+import { repositoryActions } from '../../scenes/Dashboard/actions';
+
+import styles from './styles.module.scss';
+
+export class Overview extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            activeIndex: -1
+        };
+    }
+
+    componentDidMount() {
+        const { actions } = this.props;
+        actions.fetchRepositories({
+            limit: '4',
+            filterWord: ''
+        });
+    }
+
+  handleActivityState = (e, titleProps) => {
```

Comment:
Something with indentation and code formatting, please check prettier and linter commands on your local machine and if find any issue ask @alexandrtovmach